lib/watir/locators/element/selector_builder.rb

```diff
@@ -54,9 +54,9 @@ def build(selector)
           inspect = selector.inspect
           return given_xpath_or_css(selector) if selector.key?(:xpath) || selector.key?(:css)
 
-          built = build_wd_selector(selector)
-          Watir.logger.debug "Converted #{inspect} to #{built}"
-          built
+          build_wd_selector(selector).tap do |built|
```

Comment:
🤷‍♂️

Reply:
I know, this goes against my preference of not unnecessarily using blocks.